parm/config/config.post

```diff
@@ -41,6 +41,6 @@ else
 fi
 
 export GRIBVERSION='grib2'
-export SENDCOM="YES"
+export SENDCOM=${SENDCOM:-"YES"}
```

Comment:
Consistent with @KateFriedman-NOAA's comment, this change should be reverted.

Reply:
I've reverted this change, but it did not originate in this PR. It was originally made in develop and applied to sprint when we merged develop into it. A later PR then undid that particular change in develop, but for some reason it wasn't applied in future merges of develop.

@aerorahul We keep seeing this sort of thing with squashes. Not here, but I think we need to revisit that change.